src/ngraph_encapsulate_clusters.cc

```diff
@@ -482,7 +486,27 @@ Status EncapsulateClusters(Graph* graph, int graph_id) {
     graph->RemoveNode(node);
   }
 
-  // Pass 7 (optional, only run if environment variable
+  // Pass 7: Insert to function library
+  // Note: We loop over cluster_indices_for_this_rewrite and not all the
+  // contents of ClusterManager
+  for (const auto& cluster_idx : cluster_indices_for_this_rewrite) {
+    // The transformation happening inside this loop is:
+    // graphdef --> graph --> functiondef
+    // NGraphClusterManager::GetClusterGraph(cluster_idx)-->subgraph-->fdef
+    // TODO: whats the right flib to use in sgraph's constructor?
+    Graph subgraph(graph->flib_def());
```

Comment:
Should we use OpRegistry::Global()
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/graph/graph.h#L458

Reply:
As discussed, doesn't matter as per my current understanding